docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -0,0 +1,176 @@
+* Days: Number of validity days, ie 3y = 1068.
+
+CAUTION: Remember that after the expiration date of your Certificate of Authority is passed, every certificate generated by it will not be valid anymore.
+
+Profile
+^^^^^^^
+
+Creation of a profile:
+
+* Name: MyClientProfile
+* Ca: The certificate of authority you created earlier
+* Crl path: Certification revokation list path, a file will be created when you fill this.
+* Validity: Number of validity days, ie 3y = 1068
+* Key type, size and digest: we advise to use the following RSA, 2048, sha1
+* Key Usage: Empty 
+* Extended Key Usage: 
+** Server certificate authentication profile: TLS Web Server Authentication, 
+** Client certificate authentication profile: TLS Web Client Authentication, 
+
+Information about Key Usage and Extended Key Usage are available from RFC5280, parts 4.2.1.3 and 4.2.1.12.
+
+* The P12 mail setup is mandatory at least for the server authentication profile. This will format the mail send with your certificate if you use *send certificate* button, to send it manually. 
+** If your mail alert are working with PacketFence you should use the following:
+*** P12 smtp server: 127.0.0.1
+*** Tick P12 mail password
+
+For the header, from, subject and footer this part is the customization of the mail, it is configurable as you want it for your company. An example of value would be:
+
+* Form: pki@packetfence.pki
+* Subject: Certificate for EAP-TLS connection
+* Header: Below you will find your certificate and it's password to install it.
+* Footer: This email has been generate automatically, please do not reply.
+
+RADIUS server authentication certificate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Generate a certificate from the profile you defined as the server authentication profile. This certificate will need to be installed and configured as the certificate for authenticate RADIUS. Also it will be necessary to install and trust the CA on PacketFence.
+
+CAUTION: We advise to use a long validity for your RADIUS server certificate to avoid renew it often.
+
+Client authentication certificate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Generate a certificate from the profile you defined as the client authentication profile, if you generate this certificate manually you will need to use the *download certificate* or *send certificate* buttons. When the certificate is generate via PacketFence, the certificate is automatically integrated into the profile to configure the WiFi network.
+
+Revokation process
+^^^^^^^^^^^^^^^^^^
+
+We have two ways to check if a certificate is valid, you can either use OCSP to interegote the PKI if the certificate is still valid, or use the revokation lists(CRL). However since the OCSP is implemented we advise you to use this method.
+OSCP is scalable, his con would be that one request by certificate is send to the PKI to verify if this certificate is still valid.
```

Comment:
"Use it's when refering to the protocol"  -> Its, not it's

Also, "its con would be.." doesn't make sense.